### DISCUSSION TITLE
Feedback for "argon" skin for taiko

### DISCUSSION THREAD

**Discussion by hongaaronc** [ppy/osu#21425] [Ideas]
Not sure if I should file these as separate discussions for each piece of feedback I have, but I'll just put it in this one thread for now and split it up if people prefer me to for tracking later.

Hope this feedback helps. I realize that this feedback is probably coming in much later than it could have, but I guess it's probably better for me to mention it late than never. Also I apologize in advance if some of this stuff I'm commenting on is just temp stuff.

### The new hitsounds

While the new hitsounds fit well into the general Argon/lazer soundscape aesthetic, In my opinion, they don't feel great for taiko.

Compared to the more familiar normal sampleset hitsounds from the Classic skin, it feels difficult to distinguish and less punchy in comparison. The don sound feels a bit too heavily high-pass filtered, and lacks a deep body to the sound. While it is distinguishable from the kat sounds, there isn't nearly as much contrast between don and kat as there was in the Classic skin.

I also feel like the hitsounds in taiko should be a bit more "diagetic" per se, compared to other game modes. Since taiko mapping has much heavier improv usage compared to other modes, having the taiko drum sound like an actual drum instrument that could conceivably be an actual part in the song is probably more important than fitting in with other Argon/lazer sounds.

And this isn't really unique to the Argon skin - I think this was also the case with the Triangles skin, but I haven't really thought to leave feedback until now.

### The arrow overlays

The white arrows on every note seems sort of redundant in my opinion. If I had to guess what they are trying to imply - I'm guessing it's that the notes scroll to the left - but that seems kind of obvious by just seeing them in motion.

This also looks a bit odd for the hit animation, where the notes get blasted off the playfield and move with a more physics-like behavior, but the arrows stay unnaturally upright in terms of rotation.

I think the Triangles skin had something cool with the solid or hollow circle based on don/kat. I liked how they are more agnostic to rotation/directionality.

### The judgement line graphic

The shading of the judgement line draws a lot of attention to the barline part of it, and less attention toward the circle part of it. This feels kind of weird since barlines are pretty infrequent compared to notes, and not necessarily indicative of a hitobject.

It might be a bit easier to read if these two element had more similar opacity.

### The InputDrum element
The new InputDrum element looks cool, but it seems a bit odd that the glow colors when an input is shown seems hue-shifted compared to the color of the actual notes? I think some hue shift is reasonable between this element and the color of the hitobjects, but this seems to be far enough apart where it feels like there is a disconnect and they don't feel correlated anymore.

**Comment by peppy** [comment]
Thanks for your feedback!
